### ISSUE TITLE
Allow option to raise SuspiciousOperation or PermissionDenied on low score

### ISSUE THREAD

**Issue by ghost** [kbytesys/django-recaptcha3#11]
IMHO I don't think telling a bad actor their reCAPTCHA score is a security best practice, as happens now in fields.py. A configurable option to optionally throw SuspiciousOperation and/or PermissionDenied instead, with a customizable message, would be preferable in a lot of use cases, and definitely not inform the bad actor of their score, unless say DEBUG = True.

**Comment by ghost** [comment]
If I get a sec I'll submit a PR for this. I gotta do it in my use case anyway.

**Comment by kbytesys** [comment]
Yes I agree. If you would like to submit a PR about this topic I will be very happy.

**Comment by Bishwas-py** [comment]
You can check this article dude: [Install repactcha in Django site
](https://blog.webmatrices.com/django-recaptcha-v3/)
